src/integration-test/data/test-data.ts

```diff
@@ -146,11 +146,13 @@ export function createDefendant (type: PartyType, hasEmailAddress: boolean = fal
 
   switch (type) {
     case PartyType.INDIVIDUAL:
-      defendant.name = 'Rose Smith'
+      defendant.firstName = 'Rose'
```

Comment:
Would you want to add title here as well?

Reply:
I was thinking it would be ok not to test for title being optional, since I test for its presence in almost all of the unit tests. Lemme think

Reply:
added it to individual but not sole trader, so we still have a journey checking that everything is fine if title is not provided

Reply:
(ideally I'd add two different journeys for both, but I think this is an ok compromise :D)

Reply:
That's fine.